resources/assets/js/markdown-editor/css/style.css

```diff
@@ -244,4 +244,23 @@
     color: inherit;
 }
 
+.UserTagger .highlighted .UserTagger--username {
```

Comment:
can you rename to `ark-user-tagger` or similar? to keep it in line with other css classes that we use